requirements.txt

```diff
@@ -0,0 +1,3 @@
+python-terraform<=0.10.1
+hub<=2.0
```

Comment:
which library is providing the `git` module?

Reply:
woops..